examples/ui-demo/src/components/small-cards/SolanaCard.tsx

```diff
@@ -19,6 +19,8 @@ import Image from "next/image";
 import { Badge } from "./Badge";
 
 export const SolanaCard = () => {
+  const [useSponsorship, setUseSponsorship] = useState(false);
```

Comment:
(Nit) I'd suggest naming something like `sponsorshipEnabled`. Names that start with `use` look like React hooks.

Reply:
Ah, fair enough